Submodule deps/fmt contains modified content
diff --git a/deps/fmt/CMakeLists.txt b/deps/fmt/CMakeLists.txt
index f21cf456..f8fe4a4b 100644
--- a/deps/fmt/CMakeLists.txt
+++ b/deps/fmt/CMakeLists.txt
@@ -214,7 +214,7 @@ endif ()
 
 target_compile_features(fmt INTERFACE ${FMT_REQUIRED_FEATURES})
 
-target_include_directories(fmt PUBLIC
+target_include_directories(fmt SYSTEM PUBLIC
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:${FMT_INC_DIR}>)
 
Submodule deps/spdlog contains modified content
diff --git a/deps/spdlog/CMakeLists.txt b/deps/spdlog/CMakeLists.txt
index b9694655..1d2dd8c9 100644
--- a/deps/spdlog/CMakeLists.txt
+++ b/deps/spdlog/CMakeLists.txt
@@ -150,7 +150,7 @@ endif()
 add_library(spdlog::spdlog ALIAS spdlog)
 
 target_compile_definitions(spdlog PUBLIC SPDLOG_COMPILED_LIB)
-target_include_directories(spdlog PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>"
+target_include_directories(spdlog SYSTEM PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>"
                                          "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>")
 target_link_libraries(spdlog PUBLIC Threads::Threads)
 spdlog_enable_warnings(spdlog)

Submodule deps/glfw contains modified content
diff --git a/deps/glfw/src/CMakeLists.txt b/deps/glfw/src/CMakeLists.txt
index a409459b..417d2ed3 100644
--- a/deps/glfw/src/CMakeLists.txt
+++ b/deps/glfw/src/CMakeLists.txt
@@ -100,10 +100,10 @@ else()
 endif()
 
 target_compile_definitions(glfw PRIVATE _GLFW_USE_CONFIG_H)
-target_include_directories(glfw PUBLIC
+target_include_directories(glfw SYSTEM PUBLIC
                            "$<BUILD_INTERFACE:${GLFW_SOURCE_DIR}/include>"
                            "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>")
-target_include_directories(glfw PRIVATE
+target_include_directories(glfw SYSTEM PRIVATE
                            "${GLFW_SOURCE_DIR}/src"
                            "${GLFW_BINARY_DIR}/src"
                            ${glfw_INCLUDE_DIRS})

Submodule deps/glew contains modified content
diff --git a/deps/glew/CMakeLists.txt b/deps/glew/CMakeLists.txt
index 5bfdcb2..9932675 100644
--- a/deps/glew/CMakeLists.txt
+++ b/deps/glew/CMakeLists.txt
@@ -120,7 +120,7 @@ endif()
 
 foreach(GLEW_TARGET ${GLEW_TARGETS})
 	target_compile_definitions(${GLEW_TARGET} PUBLIC ${DEFINITIONS})
-	target_include_directories(${GLEW_TARGET} PUBLIC ${INCLUDE_DIR})
+  target_include_directories(${GLEW_TARGET} SYSTEM PUBLIC ${INCLUDE_DIR})
 	target_link_libraries(${GLEW_TARGET} ${LIBRARIES})
 	set_target_properties(${GLEW_TARGET} PROPERTIES VERSION ${GLEW_VERSION})
 endforeach()

Submodule deps/yaml-cpp contains modified content
diff --git a/deps/yaml-cpp/CMakeLists.txt b/deps/yaml-cpp/CMakeLists.txt
index b230b9e..7432cce 100644
--- a/deps/yaml-cpp/CMakeLists.txt
+++ b/deps/yaml-cpp/CMakeLists.txt
@@ -1,6 +1,8 @@
 # 3.5 is actually available almost everywhere, but this a good minimum
 cmake_minimum_required(VERSION 3.4)
 
+cmake_policy(SET CMP0077 NEW)
+
 # enable MSVC_RUNTIME_LIBRARY target property
 # see https://cmake.org/cmake/help/latest/policy/CMP0091.html
 if(POLICY CMP0091)
@@ -79,7 +81,7 @@ set_property(TARGET yaml-cpp
     CXX_STANDARD_REQUIRED ON)
 
 target_include_directories(yaml-cpp
-  PUBLIC
+SYSTEM PUBLIC
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
   PRIVATE
@@ -93,9 +95,6 @@ endif()
 
 target_compile_options(yaml-cpp
   PRIVATE
-    $<${not-msvc}:-Wall -Wextra -Wshadow -Weffc++ -Wno-long-long>
-    $<${not-msvc}:-pedantic -pedantic-errors>
-
     $<$<AND:${backport-msvc-runtime},${msvc-rt-mtd-static}>:-MTd>
     $<$<AND:${backport-msvc-runtime},${msvc-rt-mt-static}>:-MT>
     $<$<AND:${backport-msvc-runtime},${msvc-rt-mtd-dll}>:-MDd>
Submodule deps/assimp contains modified content
diff --git a/deps/assimp/CMakeLists.txt b/deps/assimp/CMakeLists.txt
index dcafb649f..4c60fe18a 100644
--- a/deps/assimp/CMakeLists.txt
+++ b/deps/assimp/CMakeLists.txt
@@ -315,6 +315,7 @@ IF (ASSIMP_UBSAN)
   SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=undefined -fno-sanitize-recover=all")
   SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=undefined -fno-sanitize-recover=all")
 ENDIF()
+string(REPLACE "-Wall" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 
 INCLUDE (FindPkgMacros)
 INCLUDE (PrecompiledHeader)
diff --git a/deps/assimp/code/CMakeLists.txt b/deps/assimp/code/CMakeLists.txt
index 55538d965..456f72b5b 100644
--- a/deps/assimp/code/CMakeLists.txt
+++ b/deps/assimp/code/CMakeLists.txt
@@ -1105,7 +1105,7 @@ ENDIF (ASSIMP_BUILD_NONFREE_C4D_IMPORTER)
 ADD_LIBRARY( assimp ${assimp_src} )
 ADD_LIBRARY(assimp::assimp ALIAS assimp)
 
-TARGET_INCLUDE_DIRECTORIES ( assimp PUBLIC
+TARGET_INCLUDE_DIRECTORIES ( assimp SYSTEM PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/../include>
   $<INSTALL_INTERFACE:include>
